pull-retag-push-images.sh

```diff
@@ -10,12 +10,15 @@ yum install docker -y
 systemctl enable docker
 systemctl start docker
 
+# Wait for Docker to start...
+sleep 10
```

Comment:
Can we sleep until docker API is up?

Reply:
I've removed that bit, it doesn't seem to be really needed.